src/System/Hapistrano/Commands.hs

```diff
@@ -245,7 +245,7 @@ instance Command GitFetch where
   renderCommand (GitFetch remote) = formatCmd "git"
     [ Just "fetch"
     , Just remote
-    , Just "+refs/heads/*:refs/heads/*" ]
+    , Just "+refs/heads/\\*:refs/heads/\\*" ]
```

Comment:
It's not clear to me this part. Would you mind explain it, please?

Reply:
I think the explanation about zsh failing due to globbing is clear in the answer that @nebtrx linked in #84: https://stackoverflow.com/questions/15892624/how-to-config-gerrit-refs-in-zsh/15893007#15893007

Basically, `zsh` fails because hapistrano tries to execute:
```
$ git fetch origin +refs/heads/*:refs/heads/*
```
In `zsh` the glob fails in that case unless you escape the `*` properly. In this case, I'm escaping the commands adding a back slash before `*`, I had to add a double back slash `\\` because `\*` is not a valid string in Haskell:

```haskell
repl> putStrLn "\*"

<interactive>:4:12: error:
    lexical error in string/character literal at character '*'
```